app/mailers/purchase_notifier.rb

```diff
@@ -22,7 +22,14 @@ def product_order_notification(order_detail, recipient)
   def order_notification(order, recipient)
     @order = order
     attach_all_icals_from_order(@order)
-    subject = text("views.purchase_notifier.order_notification.subject")
+    product_count = order.products.count
+
+    subject = text(
+      "views.purchase_notifier.order_notification.subject",
+      product: product_count == 1 ? order.products.pick(:name) : nil,
+      count: product_count,
```

Comment:
Is this value used?

Reply:
it's used to tell which locale to use, either `one` or `other`